### DISCUSSION TITLE
Is there a method to search the url of interfaces

### DISCUSSION THREAD

**Discussion by cavernBuilderZYX** [hoppscotch/hoppscotch#4648] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/23382a6b-a250-45ae-afcd-a6a3e7992a6b)

For example, I have a interface in the current collection with keyword "purchase" in the url, but it can't be search

**Comment by SanskritiHarmukh** [comment, marked as answer]
Hi there, the current collection search functionality in Hoppscotch is scoped to matching keywords against the names of parent collections, child collections, and requests. It does not support indexing or querying the URLs of requests. As a result, keywords present in the request URL (e.g., "purchase") will not yield any search results unless the keyword is explicitly part of the collection or request name.
